package.json

```diff
@@ -30,7 +30,8 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.2.2",
     "mercurius": "^13.0.0",
-    "prettier": "^2.8.8"
+    "prettier": "^2.8.8",
+     "dotenv": "^16.0.3"
```

Comment:
Please take a note why this library is used for.

Reply:
please format correctly